modules/tinymce/CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - The `fontsizeinput` increase and decrease size buttons now work on TinyMCE mobile. #TINY-9725
-- The TinyMCE editor toolbar is now accessible for all editor widths; it no longer collapses into an inaccessible vertical line at any presented editor width. #TINY-9646
+- The TinyMCE editor toolbar is now accessible for all screen widths; it no longer collapses into an inaccessible vertical line when the screen is scrolled horizontally. #TINY-9646
```

Comment:
6.4.1 is already released so a bit odd to change a published changelog. But I guess if you read up on changes for the next release it would be useful.

Reply:
> 6.4.1 is already released so a bit odd to change a published changelog.

Agreed. The back-story is that the above corrections were found and corrected too close to publication to be be corrected before publication.

They were, however, found just in time to correct the somewhat-more-public version of the Changelog, published as part of the [Docs-suite](https://tiny.cloud/docs/tinymce/6/changelog/), before said somewhat-more-public version of the Changelog was updated.

So this PR is, in effect, a, so-far singular, example of bringing the GitHub-hosted changelog into sync with the Docs-hosted changelog (rather than the other way round, which is our standard way of doing things).